### ISSUE TITLE
High-contrast mode: tag field shouldn’t override background

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#7454]
This is a bug specific to [Windows high-contrast mode](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/). cc @kbayliss who did the testing for this.

> Although the styling doesn't actually look too bad here, it is not correct. The field should be styled the same as other fields.

![7454](https://user-images.githubusercontent.com/877585/129972161-bf89ab4e-46a1-40c8-95ee-727c80e2158b.png)

---

I would expect we’d want the field with a border of the appropriate color, dark background, and the tag to use an active element’s styles.

**Comment by Desaiakshata** [comment]
I would like to work on this issue

**Comment by Desaiakshata** [comment]
I have an error on running `npm run lint:css`  : Expected a dollar-variable (e.g. $var) to be used as a media feature value   scss/media-feature-value-dollar-variable. Here is the code change: `@media (forced-colors:active) {
    .field-content .tagit .tagit-choice, .tagit, .tagit .tagit-new .ui-widget-content {
        forced-color-adjust: none;
        box-shadow: inset 1000px 0 0 0 #000;
        color: #fff
    }`

**Comment by lb-** [comment]
Fixed via 22cdd89b682c2c5e181002038bcaf8daf1db63b7 #7499
